OPHD/Map/MapOffset.h

```diff
@@ -8,3 +8,36 @@ struct MapOffset
 	NAS2D::Vector<int> xy;
 	int z;
 };
+
```

Comment:
Style guidelines: double spaces between methods and functions. Otherwise I'm not seeing anything blocking here. Would like to see some feedback from @DanRStevens before merging.

Reply:
Ok will fix. I add two more commits to actually use the new operators. Oh and we some how missed updating calls to the new MapViewState:moveView from mMapView::moveView in MapViewState::onKeyDown